MANUAL.txt

```diff
@@ -597,8 +597,8 @@ Reader options {.options}
 
 `--metadata-file=`*FILE*
 
-:   Read metadata from the supplied YAML (or JSON) file.
-    This option can be used with every input format, but string
+:   Read metadata from the supplied YAML (or JSON) file. Several file may be
+    provided. This option can be used with every input format, but string
```

Comment:
typo: should be `files`.

or better yet, usually we have the wording: `This option can be used repeatedly`